mocha/specs/test.spec.js

```diff
@@ -0,0 +1,124 @@
+'use strict'

```

Comment:
A test.png image was committed. Do you think it yould be added to gitignore, so test report related data is not tracked?